src/utils/verseObjects.js

```diff
@@ -18,26 +18,26 @@ export const tokenizeVerseObjects = (verseObjects) => {
     }
     sentenceCharLength += word.text.length;
     occurrences[word.text]++;
-    tokens.push(new Token({
```

Comment:
Any reason why `new Token` was removed? is it not needed anymore?

Reply:
It was duplicate code. 
If you check the logic you will see it was unnecessarily created just to be re-created.
When happening a couple of hundred times it causes performance issues.

Reply:
Sounds good. Just wanted to double check.